Source/menu.cpp

```diff
@@ -58,7 +58,7 @@ bool InitMenu(_selhero_selections type)
 	if (type == SELHERO_PREVIOUS)
 		return true;
 
-	success = StartGame(type != SELHERO_CONTINUE, type != SELHERO_CONNECT);
+	success = StartGame(true, type != SELHERO_CONNECT);

```

Comment:
Was this intentional?

Reply:
Yes, but I made the mistake of trusting JetBrain's inter